src/ui.form-input-money/UInputMoney.vue

```diff
@@ -283,4 +281,24 @@ function onBlur() {
 function onInput(value) {
   nextTick(() => emit("input", value));
 }
+
+defineExpose({
+  /**
+   * Reference to the underlying input element inside UInput.
+   * @property {HTMLInputElement}
+   */
+  input,
+
+  /**
+   * The raw, unformatted value of the input.
+   * @property {String | Number}
```

Comment:
Does this format handled and shows well in storybook?

Reply:
![image](https://github.com/user-attachments/assets/bdf9ea6d-e3a0-4a2e-9618-b94ab6a9e5c2)

Yes, it looks fine